R/travis-s3.R

```diff
@@ -11,7 +11,7 @@ travis_no_attr <- function(x) {
 new_travis <- function(x, attr, subclass) {
   attr[["names"]] <- names(x)
   attributes(x) <- attr
-  structure(x, class = c(paste0("travis_", subclass), "travis"))
+  suppressWarnings(structure(x, class = c(paste0("travis_", subclass), "travis")))
```

Comment:
What does the `suppressWarnings()` suppress?

Reply:
A warning related to the `structure()´ call. When being removed, the return of the function changes and leads to errors downstream. So I decided to suppress it and save me headaches.

Reply:
Under what conditions does `structure()` warn?